modular_skyrat/modules/walk_curtains/curtains.dm

```diff
@@ -0,0 +1,21 @@
+/obj/structure/curtain
+	/// whether curtains will toggle their density when toggled
+	var/toggle_density = FALSE
+
+/obj/structure/curtain/toggle()
```

Comment:
If we're gonna override a proc like this can we add a comment over in the original define of this so that people know to find the override here on the offchance we need to adjust this down the line?